### PR TITLE
Update versions.tf

### PR DESCRIPTION
**Pull request by neel-gala**
Need a support for latest version. We have tested it on the terraform version 1.0.6 and it was working fine for us.

### CODE REVIEW

**Review by samen93 — COMMENTED** [review]
looks good to me

File: versions.tf [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # ----------------------------------------------------------------------------------------------------------------------
 
 terraform {
-  required_version = ">= 0.12.20, < 0.16.0"
+  required_version = ">= 0.12.20, < 1.0.6"
```
Comment (mariux) [inline versions.tf:6]: Hey, thanks for your contribution.. we can assume that the module will support all major 1.x releases if hashicorp will stick to semantic versioning in their major releases.

```suggestion
  required_version = ">= 0.12.20, < 2.0"
```

for a release we would also need to update tests to run with the new version and ofc update docs ;) but we can already merge this once the version constraint is fixed from my point of view.

cc @soerenmartius

**Review by mariux — CHANGES_REQUESTED** [review]

File: versions.tf (hunk 1 above)
Comment (neel-gala) [reply versions.tf]: makes sense.

**Review by mariux — APPROVED** [review]
lgtm

**Comment by soerenmartius** [comment]
Hi @neel-gala , thanks for providing this! I've implemented support for Terraform `v1.x` in https://github.com/mineiros-io/terraform-aws-lambda-function/releases/tag/v0.5.0 (including tests, documentation, etc.). You should be able to go ahead with the new release.

Thanks and have a good weekend!

**Pull request closed without merging**